server/manager/src/main/java/org/apache/accumulo/manager/upgrade/Upgrader9to10.java

```diff
@@ -133,6 +141,42 @@ public void upgradeZookeeper(ServerContext context) {
     createScanServerNodes(context);
   }
 
+  private static final AtomicBoolean aclErrorOccurred = new AtomicBoolean(false);
+
+  private void validateACLs(ServerContext context) {
+
```

Comment:
```suggestion

  private void validateACLs(ServerContext context) {

    final AtomicBoolean aclErrorOccurred = new AtomicBoolean(false);
```

Seems like this only used in the method.  Wondering if could be declared in the method.